plugins/woocommerce-admin/client/homescreen/mobile-app-modal/components/MobileAppLoginStepper.tsx

```diff
@@ -0,0 +1,151 @@
+/**
+ * External dependencies
+ */
+import React, { useState, useEffect } from '@wordpress/element';
+import { Button } from '@wordpress/components';
+import { sprintf, __ } from '@wordpress/i18n';
+import { Stepper, StepperProps } from '@woocommerce/components';
+
+/**
+ * Internal dependencies
+ */
+import { SendMagicLinkButton, SendMagicLinkStates } from './';
+import { getAdminSetting } from '~/utils/admin-settings';
+import { MobileAppInstallationInfo } from '../components/MobileAppInstallationInfo';
+import { MobileAppLoginInfo } from '../components/MobileAppLoginInfo';
+
+export const MobileAppLoginStepper = ( {
```

Comment:
The height needs to be adjusted so that we don't have a scrollbar, I feel the vertical scrolling on this gives it an unpolished feel

It's slightly complicated because it seems that the height on the two steps are different? I think it would be best if they have the same height so that there's no layout change between the steps